src/Connect/index.tsx

```diff
@@ -86,6 +87,15 @@ const ConnectScreen: FunctionComponent = () => {
     listItems.push(reportAnIssue)
   }
 
+  if (displayCallbackForm) {
```

Comment:
@devinjameson I punted on adding the sections since that seems to be closer to your plate, but let me know if that fits in this scope and I can take on that(or we can 🍐 ) 😄

Reply:
Sounds good! We can do it in another pass :)